### DISCUSSION TITLE
How can we reset the input value?

### DISCUSSION THREAD

**Discussion by kozak-iz-kh** [andreaswilli/react-verification-input#4] [Q&A] [answered]

**Comment by andreaswilli** [comment]
Hi, have a look at #2 which allows you to access the input element.
Might add an easier option at some point. (Also, I'm happy to review your PR if you want to implement it yourself 😃).

**Comment by kozak-iz-kh** [comment]
Thanks, I hope to get back to it later.

**Comment by andreaswilli** [comment, marked as answer]
Starting from version `2.0.0` the value can be modified from the outside via the `value` prop:

```jsx
const [value, setValue] = useState("");

// ...

<VerificationInput value={value} onChange={setValue} />
<button onClick={() => setValue("")}>reset</button>
```
